.github/workflows/release.yaml

```
@@ -0,0 +1,30 @@
+# Copyright 2023 Adevinta
+
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+permissions:
+  contents: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
+      - uses: goreleaser/goreleaser-action@v5
```

Add a descriptive name to the step.

```suggestion
      - name: Run "goreleaser"
        uses: goreleaser/goreleaser-action@v5
```